### DISCUSSION TITLE
[FEATURE]: Sales Exchange Rate Request

### DISCUSSION THREAD

**Discussion by gokhur** [etsy/open-api#540] [Ideas]
I can get exchange rate for each sale from the Etsy Payment Sales report( as CSV) in our store panel (options >> download data) so I have to download from there manually, is there any chance to get exchange rate from the APIs?

**Comment by etsyachristensen** [comment]
At this time there is no plan to add this to the OpenAPI. I will add it to our list of requests though.

**Comment by etsyachristensen** [comment]
@gokhur I have moved this issue to the discussions section as an Idea so that we can more easily track this as a future feature request.

**Comment by gokhur** [comment]
Thank you very much :) actually It is really important data to create
invoices because we have to create invoices in Turkish Liras rate because
etsy is transfering as Turkish Liras

20 Tem 2022 Çar, saat 19:37 tarihinde Andrew Christensen <
***@***.***> şunu yazdı:

> @gokhur <https://github.com/gokhur> I have moved this issue to the
> discussions section as an Idea so that we can more easily track this as a
> future feature request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/etsy/open-api/discussions/540#discussioncomment-3191033>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFHVXEDLGNPTHZGA64SM2LVVATN7ANCNFSM54ELDS5Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
--
Gökhür ARIKBOĞA
